mesh/submesh/submesh.hpp

```diff
@@ -140,6 +152,43 @@ class SubMesh : public Mesh
       return parent_vertex_ids_;
    }
 
+   /**
+    * @brief Get the submesh element corresponding to a parent element. -1 == not present.
```

Comment:
Should it be `< 0 == not present`? Can there be negative values not equal to `-1`? If not, change `pe < 0` to `pe == -1`?